src/framing/mod.rs

```diff
@@ -800,7 +800,6 @@ impl Codec for MLSPlaintextCommitContent {
 
 pub struct MLSPlaintextCommitAuthData {
     pub confirmation_tag: Vec<u8>,
-    pub signature: Vec<u8>,
```

Comment:
As of yet, the signature is still part of the auth data. See https://messaginglayersecurity.rocks/mls-protocol/draft-ietf-mls-protocol.html#name-group-state.

Reply:
I think you should refresh the page...

Reply:
Argh. Now I'm wondering how long I've been staring at an outdated version.

In any case, good to merge, then.

Reply:
Part of my confusion, I think, was because that particular change is not part of the PR pointed to in #428 (the mls-protocol issue).

Reply:
Yes, this is the very latest spec change. I implemented it to validate the spec change early on, obviously there are other pending changes we need to address.